mutalyzer/website/templates/description-extractor.html

```diff
@@ -69,8 +120,38 @@
   {% if not errors %}
     <hr>
 
-    <h4>Genomic description</h4>
-    <p><code>g.{{ description }}</code></p>
+    <h4>Input</h4>
+    <table class="table">
+      <thead>
+        <tr>
+          <th>Field</th>
+          <th>Value</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+        {% if method == 'raw_method' %}
+          <td>Reference sequence</td>
+          <td><code>{{ reference_sequence|short(91) }}</code></td>
+        </tr>
+        <tr>
+          <td>Sample sequence</td>
+          <td><code>{{ variant_sequence|short(91) }}</code></td>
+        {% endif %}
+        {% if method == 'refseq_method' %}
+          <td>Reference accession number</td>
+          <td><code>{{ reference_accession_number }}</code></td>
+        </tr>
+        <tr>
+          <td>Sample accession number</td>
+          <td><code>{{ sample_accession_number }}</code></td>
+        {% endif %}
+        </tr>
+      </tbody>
+    </table>
+    
+    <h4>Description</h4>
+    <p><code><a href="/description?description={{ description }}">{{ description|unicode|too_long(115) }}</a></code></p>
```

Comment:
I don't see any problem with recalculating the description. Could you elaborate why you want to prevent it?

Note that opening the description in a new window with javascript would also be subject to the 2K characters limitation.